packages/account/src/Components/trading-assessment/trading-assessment-existing-user.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Formik, Form } from 'formik';
+import {
+    FormSubmitButton,
+    Text,
+    Icon,
+    Button,
+    Modal,
+    DesktopWrapper,
+    MobileDialog,
+    MobileWrapper,
+} from '@deriv/components';
+import { generateValidationFunction, getDefaultFields, isMobile } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { trading_assessment_form_config } from 'Configs/trading-assessment-config';
+import TradingAssessmentRadioOption from './trading-assessment-radio-buttons';
+import TradingAssessmentDropdownOption from './trading-assessment-dropdown';
+
+const TradingAssessmentForm = ({ assessment_questions }) => {
+    const [current_question_index, setCurrentQuestionIndex] = React.useState(0);
+    const [is_next_button_enabled, setIsNextButtonEnabled] = React.useState(false);
+    const [current_question, setCurrentQuestion] = React.useState({});
```

Comment:
Question: is it possible to combine `current_question` and `current_question_index` into one state with object? They seem to be updating simultaneously.

Reply:
Done